GitUI/SpellChecker/WordAtCursorExtractor.cs

```diff
@@ -1,41 +1,83 @@
-﻿using System.Text;
+﻿using System;
+using System.Text;
 
 namespace GitUI.SpellChecker
 {
     internal class WordAtCursorExtractor : IWordAtCursorExtractor
     {
         public string Extract(string text, int cursor)
         {
+            var startIndex = FindStartOfWord(text, cursor);
+            return startIndex < 0 ? string.Empty : text.Substring(startIndex, cursor - startIndex + 1);
+        }
+
+        public (int start, int length) FindWord(string text, int cursor)
```

Comment:
I find the proposed name somewhat misleading - we don't find a word, rather word's bounds.
I'm struggling to propose a good name atm, perhaps something like `CalculateWordBounds`?